lib/ui/menu_bottom_sheet.dart

```diff
@@ -14,176 +14,144 @@ class MenuBottomSheet extends StatelessWidget {
-                "https://www.pola-app.pl/m/method"
-              ),
-              _WebViewItem(
-                "O Klubie Jagiellońskim",
-                analytics,
-                AnalyticsAboutRow.aboutKJ,
-                "https://klubjagiellonski.pl/o-klubie-jagiellonskim/"
-              ),
-              _WebViewItem(
-                "Zespół",
-                analytics,
-                AnalyticsAboutRow.team,
-                "https://www.pola-app.pl/m/team"
-              ),
-              _WebViewItem(
-                "Partnerzy",
-                analytics,
-                AnalyticsAboutRow.partners,
-                "https://www.pola-app.pl/m/partners"
-              ),
-              _WebViewItem(
-                "Przyjaciele Poli",
-                analytics,
-                AnalyticsAboutRow.polasFriends,
-                "https://www.pola-app.pl/m/friends"
-              ),
-              _ExternalUrlItem(
-                "GitHub",
-                analytics,
-                AnalyticsAboutRow.github,
-                "https://github.com/KlubJagiellonski/pola-flutter"
-              ),
-              _ExternalUrlItem(
-                "Oceń Polę",
-                analytics,
-                AnalyticsAboutRow.rateUs,
-                Platform.isIOS 
-                  ? "https://apps.apple.com/app/id1038401148"
-                  : "https://play.google.com/store/apps/details?id=pl.pola_app"
-              ),
-              Row(
-                children: [
-                  Expanded(
-                      child: _ExternalUrlItem(
-                          "Facebook",
-                          analytics,
-                          AnalyticsAboutRow.facebook,
-                          "https://www.facebook.com/app.pola"
-                        )
-                  ),
-                  Expanded(
-                      child: _ExternalUrlItem(
-                          "X",
-                          analytics,
-                          AnalyticsAboutRow.twitter,
-                          "https://twitter.com/pola_app"
-                        )
-                 )
-                ],
+        child: Column(
+          children: [
+            const SizedBox(height: 13),
+            Container(
+              width: 47,
+              height: 3,
+              decoration: const BoxDecoration(
+                color: Colors.black,
+                borderRadius: BorderRadius.all(Radius.circular(1.5)),
               ),
-              Padding(
-                  padding: EdgeInsets.all(6.0),
-                  child: Align(
-                    alignment: Alignment.centerLeft,
-                    child: Text("Aplikacja Pola \n©Klub Jagielloński"),
-                  ))
-            ],
-          ),
+            ),
+            const SizedBox(height: 11),
+            MenuItemListview(analytics: analytics),
+            const SizedBox(height: 23),
+            Container(
+              width: 328,
```

Comment:
Ten kontener nie może mieć podanej szerokości. Trzeba to paddingami ograć, bo jak będziesz miał telefon innej szerokości to się rozjedzie z innymi elementami.